catalog/core/forms.py

```diff
@@ -23,12 +23,25 @@ class CatalogSearchForm(SearchForm):
     publication_end_date = forms.DateField(required=False)
     contact_email = forms.BooleanField(required=False)
     status = forms.ChoiceField(choices=STATUS_CHOICES, required=False)
+    tags = forms.CharField(required=False, widget=forms.Select(attrs={'multiple': "multiple", 'name': "tags",
+                                                                      'data-bind': "selectize: tags, selectedOptions: selectedTags, optionsCaption: 'Keywords', optionsValue: 'name', options: { create: false, load: getTagList, hideSelected: true }, value: SelectedTags"}))
+    authors = forms.CharField(required=False)                                                                  
     assigned_curator = forms.CharField(required=False)
     flagged = forms.ChoiceField(choices=ANY_CHOICES, required=False)
     is_archived = forms.ChoiceField(choices=ANY_CHOICES, required=False, label=_("Has code URL"))
 
     def no_query_found(self):
         return self.searchqueryset.filter(is_primary=True).models(Publication).all()
+        
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+        super(CatalogSearchForm, self).__init__(*args, **kwargs)
+
+    def clean(self):
+        super(CatalogSearchForm, self).clean()
+        cleaned_data = self.cleaned_data
+        cleaned_data['tags'] =  self.request.GET.getlist('tags')
```

Comment:
This looks OK but I'd prefer it if you only pass the tags themselves into the form from CatalogSearchView instead of the entire request object. Or can you circumvent this entirely via http://stackoverflow.com/questions/11190070/django-getlist ?